tests/network/localnet/conftest.py

```
@@ -549,3 +551,123 @@ def ovs_bridge_localnet_running_jumbo_frame_vms(
+@pytest.fixture(scope="module")
+def running_shared_hostname_vms(
+    shared_hostname_vm_1: BaseVirtualMachine,
+    shared_hostname_vm_2: BaseVirtualMachine,
+    shared_hostname_vm_3: BaseVirtualMachine,
+) -> list[BaseVirtualMachine]:
+    running_vms = run_vms(vms=(shared_hostname_vm_1, shared_hostname_vm_2, shared_hostname_vm_3))
+    ip_families = supported_cluster_ip_versions()
+    for vm in running_vms:
+        vmi_hostname = vm.vmi.instance.spec.hostname
+        if vmi_hostname != SHARED_HOSTNAME:
+            raise RuntimeError(f"VM {vm.name} hostname is {vmi_hostname!r}, expected {SHARED_HOSTNAME!r}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)